roles/matrix-prometheus-node-exporter/defaults/main.yml

```diff
@@ -54,3 +54,12 @@ matrix_prometheus_node_exporter_metrics_proxying_enabled: false
 # because node-exporter can't see all interfaces, etc.
 # For now, we'll live with that, until someone develops a better solution.
 matrix_prometheus_node_exporter_container_http_host_bind_port: ''
+
+# If you are supplying your own NGINX proxy but want to use the provided exporters you will have to supply an "<ip>:<port>" value for the containers to bind to on your host.
+# If matrix_prometheus_node_exporter_container_http_host_bind_port is set to just a port number, this will default to "127.0.0.1:<port>"
+# If matrix_prometheus_node_exporter_container_http_host_bind_port is set to an IP that is not  0.0.0.0 and a port, that "<ip>:<port>" value will be used
+# Otherwise this value will be empty and you will have to manually configure your NGINX config file. (If you are using the config files generated by this playbook, you will have to edit matrix-domain.conf)
+matrix_prometheus_node_exporter_matrix_nginx_proxy_not_enabled_proxy_pass_host: "{{ '127.0.0.1' + matrix_prometheus_node_exporter_container_http_host_bind_port_number_raw if not ':' in matrix_prometheus_node_exporter_container_http_host_bind_port else (matrix_prometheus_node_exporter_container_http_host_bind_port if matrix_prometheus_node_exporter_container_http_host_bind_port.split(':')[0] != '0.0.0.0' else '') }}"
+
+matrix_prometheus_node_exporter_dashboard_urls:
+  - "https://grafana.com/api/dashboards/9628/revisions/7/download"
```

Comment:
This seems unused. Looks like it got copied over from `roles/matrix-prometheus-postgres-exporter/defaults/main.yml`.

Reply:
Yep, I fatfingered when pasting and then replaced every instance of the word postgres with node